src/client/pages/HLI/HLIStadspasDetail.tsx

```diff
@@ -79,21 +80,21 @@ const displayPropsBudgets = {
 };
 
 const PHONENUMBERS = {
-  CCA: '14020',
+  CCA: '14 020',
   WerkEnInkomen: '020 252 6000',
 } as const;
 
-export default function HLIStadspas() {
+export default function HLIStadspasDetail() {
   const isPhoneScreen = usePhoneScreen();
   const appState = useAppStateGetter();
 
   const { HLI } = appState;
   const { passNumber } = useParams<{ passNumber: string }>();
   const stadspassen = useStadspassen();
 
-  const stadspas = passNumber
+  const stadspas: StadspasFrontend | undefined = passNumber
     ? stadspassen.find((pass) => pass.passNumber.toString() === passNumber)
-    : null;
+    : undefined;
```

Comment:
![Screenshot 2025-03-05 at 10 06 11](https://github.com/user-attachments/assets/05dd7cc2-cadc-4644-8d93-35a813bb6c36)
Je kunt hier gewoon

```
const stadspas = stadspassen.find(
    (pass) => pass.passNumber.toString() === passNumber
  );
```

doen.

Reply:
Wanneer je in de `useStadspassen` hook de data typed:

```
export function useStadspassen() {
  const { HLI } = useAppStateGetter();
  const { data: passBlokkadeByPasnummer } = useBlockStadspas();
  const stadspassen = (HLI.content?.stadspas || []).map((pas) => {
    const isGeblokkeerd = passBlokkadeByPasnummer?.[pas.passNumber];
    const stadspas: StadspasFrontend = { // <----------
      ...pas,
      actief: isGeblokkeerd ?? pas.actief,
    };

    return stadspas;
  });

  return stadspassen;
}
```